test/snippet/range/view/to_case.cpp

```diff
@@ -0,0 +1,40 @@
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/range/view/to_lower.hpp>
+#include <seqan3/range/view/to_upper.hpp>
+
+#include <vector>
+
+
+using namespace seqan3;
+
+int main()
+{
+
+{
+//! [to_lower]
+std::string s{"CHanGED!"};
+std::vector<std::string> sv{"CHanGED!", "unchanged"};
```

Comment:
In your example design the `!` is unchanged so it should be `{"CHanGED", "unchanged!"}` right? :nerd_face: